src/main/java/core/basesyntax/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Driver;
+import core.basesyntax.service.CarService;
+import core.basesyntax.service.DriverService;
+import java.io.IOException;
+import java.util.NoSuchElementException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("core.basesyntax");
+    private final CarService carService = (CarService) INJECTOR.getInstance(CarService.class);
+    private final DriverService driverService = (DriverService) INJECTOR
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/driverToCar.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        try {
+            Long driverId = Long.valueOf(req.getParameter("driver_id"));
+            Long carId = Long.valueOf(req.getParameter("car_id"));
+            Car car = carService.get(carId);
+            Driver driver = driverService.get(driverId);
+            carService.addDriverToCar(driver, car);
+            resp.sendRedirect(req.getContextPath() + "/car/");
+        } catch (NoSuchElementException e) {
+            req.setAttribute("message", "There is no such id, please enter valid id");
+            req.getRequestDispatcher("/WEB-INF/views/car/driverToCar.jsp").forward(req, resp);
+        }
```

Comment:
let's not catch RuntimeExceptions. Make all the validations from the jsp side. In scope of this task we may just "believe" that we will get the existing id

Reply:
Fixed